config/opal_load_platform.m4

```diff
@@ -155,13 +172,15 @@ AC_DEFUN([OPAL_LOAD_PLATFORM], [
 
                     AC_MSG_NOTICE([Platform patches applied, created stamp file ${patch_done}])
                     touch ${patch_done}
+                else
+                    AC_MSG_NOTICE([No platform patches in ${patch_dir}])
                 fi
 
             else
                 AC_MSG_WARN([Platform patches already applied, skipping. ${patch_done} can be removed to re-apply ])
             fi
-        else
-            AC_MSG_NOTICE([No platform patches in ${patch_dir}])
+          else
+            AC_MSG_NOTICE([No platform patches ${patch_dir}])
```

Comment:
here patch_dir can be empty (line: 100)
https://github.com/miked-mellanox/ompi-release/commit/d1fe82ed6179ea894a556a0fe050ed324ce006e3#diff-d062f072b6901db0f80c53d37e0c80ceR110